.github/ISSUE_TEMPLATE/DOCUMENTATION_ISSUE.yml

```diff
@@ -0,0 +1,34 @@
+name: "Documentation Issue 📚"
+description: Report issues in our documentation
+title: " "
+labels:
+  - documentation
+  - backlog
+assignees: " "
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this report! Before submitting, please make sure to [search through existing issues](https://github.com/Azure/data-product-streaming/issues) and [check the Known Issues section](/docs/EnterpriseScaleAnalytics-KnownIssues.md).
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Documentation Issue
+      placeholder: Briefly describe which document needs to be corrected and why.
```

Comment:
Consider rewording this sentence. 

"Please share the name of the document along with supporting points on why it requires an amendment."